packages/web-components/fast-components/src/avatar/index.ts

```diff
@@ -0,0 +1,68 @@
+import { attr, html, when } from "@microsoft/fast-element";
+import {
+    AvatarOptions,
+    Avatar as FoundationAvatar,
+    avatarTemplate as template,
+} from "@microsoft/fast-foundation";
+import { avatarStyles as styles } from "./avatar.styles";
+
+export class Avatar extends FoundationAvatar {
+    /**
+     * Indicates the Avatar should have an image source
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: src
+     */
+    @attr({ attribute: "src" })
+    public imgSrc: string;
+
+    /**
+     * Indicates the Avatar should have alt text
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: alt
+     */
+    @attr public alt: string;
+}
+
+export const imgTemplate = html<Avatar>`
```

Comment:
I think you'll want to add tsdocs here as `@public` for the documentation - that's where the build error is coming from.